samples/GeneratorInConsumerSolution/Directory.Build.props

```diff
@@ -1,7 +1,9 @@
 <Project>
+
+  <Import Project="$(MSBuildThisFileDirectory)../PackageSources.props"/>
   
   <PropertyGroup>
-    <LocalNuGetVersion>*</LocalNuGetVersion>
+    <LocalNuGetVersion>0.6.5-g5df32cb0b6</LocalNuGetVersion>
```

Comment:
Was this change intentional?

Reply:
Definitely not 😨

Reply:
I'll revert the whole commit anyway, then.